src/components/PostPreview/PostPreviewCard.tsx

```diff
@@ -0,0 +1,52 @@
+import './PostPreview.scss';
+
+import { Box } from '@eleven-labs/design-system';
+import classNames from 'classnames';
+import React from 'react';
+import { PropsWithChildren } from 'react';
+
+import { PostPreviewProps } from './PostPreview';
+
+const PostPreviewCard = ({
+  isHighlighted,
+  image,
+  hasMask,
+  isRelated,
+  // boxProps,
+  children,
+}: Partial<PostPreviewProps> & PropsWithChildren): React.ReactNode => {
```

Comment:
Créer une interface dédier pour ce composant
```suggestion
const PostPreviewCard: React.FC<PostPreviewCardProps> = ({
  isHighlighted,
  image,
  hasMask,
  isRelated,
  children,
  ...props
}) => {
```